packages/backend/src/tests/openPaymentsMocks.ts

```diff
@@ -0,0 +1,92 @@
+import { v4 as uuid } from 'uuid'
```

Comment:
I took this from the `open-payments` package, since I wanted to mock the results from the `open-payments`client calls. I could also export these mocks directly from the library?

Reply:
I'm good with exported mocks